src/styles/nav.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+  margin: 0;
+}
+
+.link__adress {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  margin-right: 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+  transition: 0.3s;
+  transition-property: color;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+
+  &:after {
+    position: absolute;
+    content: "";
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    bottom: 0;
+  }
+
+}
+
+.link__adress:hover {
+  color: #00acdc;
+}
+
+.nav__link:last-of-type .link__adress {
+  margin-right: 0;
+}
```

Comment:
You can avoid this specificity increasing if you'll set margins for the `li` instead of `a`. Also it's strange to set distance between siblings for this sibling children, it's a distance between what?